### PR TITLE
CORE-2232

### PR DESCRIPTION
**Pull request by findepi**
https://liquibase.jira.com/browse/CORE-2232

### CODE REVIEW

**Comment by nvoxland** [comment]
Thanks!

File: liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java [hunk 1]
```diff
@@ -235,7 +236,14 @@ public boolean isReservedWord(String tableName) {
 
     @Override
     protected String getConnectionSchemaName() {
-        return "public";
+        try {
+            String currentSchema = ExecutorService.getInstance().getExecutor(this)
+                    .queryForObject(new RawCallStatement("select current_schema"), String.class);
```
Comment (danielc576) [inline liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java]: Hi, 
This query should be: "select current_schema()"
Please see: http://www.postgresql.org/docs/8.3/static/functions-info.html
Can you please fix this ?

Thanks,
Daniel

Comment (nvoxland) [inline liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java]: I pushed a change to master for this change

**Comment by shrz** [comment]
@nvoxland 
It seems that current_schema without () doesn't work for PostgreSQL < 8.4
http://www.postgresql.org/docs/8.3/static/functions-info.html
http://www.postgresql.org/docs/8.4/static/functions-info.html

So Current Liquibase Core Release: 3.4.1 (July 28, 2015) doesn't work for PostgreSQL < 8.4.

**Pull request merged**